### PR TITLE
(DOCSP-10666) Update mongosh connection page with --port and --host

### PR DESCRIPTION
**Pull request by zach-carr**
[JIRA](https://jira.mongodb.org/browse/DOCSP-10666)
[Staged](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker/DOCSP-10666/connect)

Broke out **Local MongoDB Instance on a Non-Default Port** and **MongoDB Instance on a Remote Host** into bulleted lists to cover --host and --port options as well.

### CODE REVIEW

File: source/connect.txt [hunk 1]
```diff
@@ -39,38 +39,74 @@ This is equivalent to the following command:
...
-   To connect to a MongoDB instance running on localhost with a 
-   non-default port 28015:
+  .. example::
+  
+     To connect to a MongoDB instance running on localhost with a 
+     non-default port 28015:
+  
+     .. code-block:: sh
+  
+        mongosh "localhost:28015"
```
Comment (jwilliams-mongo) [inline source/connect.txt:53]:
```suggestion
        mongosh "mongodb://localhost:28015"
```

Comment (jwilliams-mongo) [reply source/connect.txt:53]: the commands do the same thing but it's nice to show the protocol here for completeness.

Comment (jwilliams-mongo) [reply source/connect.txt:53]: consider making this change in the `Local MongoDB Instance on Default Port` section, too.

File: source/connect.txt [hunk 2]
```diff
@@ -39,38 +39,74 @@ This is equivalent to the following command:
...
+  .. example:: 
+
+     To connect to a MongoDB
+     instance running on a remote host machine on port 28015:
+
+     .. code-block:: sh
+
+        mongosh --host mongodb://mongodb0.example.com:28015
+
+- Command-line options ``--host`` and ``--port``.
```
Comment (jwilliams-mongo) [inline source/connect.txt:100]:
```suggestion
- The command-line options ``--host`` and ``--port``.
```

File: source/connect.txt [hunk 3]
```diff
@@ -39,38 +39,74 @@ This is equivalent to the following command:
...
-.. example:: 
+- A :manual:`connection string </reference/connection-string>`.
 
-   To connect to a MongoDB
-   instance running on a remote host machine on port 28015:
+  .. example:: 
 
-   .. code-block:: sh
+     To connect to a MongoDB
+     instance running on a remote host machine on port 28015:
```
Comment (jwilliams-mongo) [inline source/connect.txt:76]:
```suggestion
     instance running on a remote host on port 28015:
```

File: source/connect.txt [hunk 4]
```diff
@@ -39,38 +39,74 @@ This is equivalent to the following command:
...
 
-      mongosh "mongodb://mongodb0.example.com:28015"
+- The command-line option ``--host``, 
+  specifying a port in the format ``<host>:<port>``.
 
-.. note:: Connecting to |service|
+  .. example:: 
+
+     To connect to a MongoDB
+     instance running on a remote host machine on port 28015:
```
Comment (jwilliams-mongo) [inline source/connect.txt:94]:
```suggestion
     instance running on a remote host on port 28015:
```

File: source/connect.txt [hunk 5]
```diff
@@ -39,38 +39,74 @@ This is equivalent to the following command:
...
+     .. code-block:: sh
+
+        mongosh --host mongodb://mongodb0.example.com:28015
+
+- Command-line options ``--host`` and ``--port``.
+
+  .. example:: 
+
+     To connect to a MongoDB
+     instance running on a remote host machine on port 28015:
```
Comment (jwilliams-mongo) [inline source/connect.txt:105]:
```suggestion
     instance running on a remote host on port 28015:
```

**Comment by zach-carr** [comment]
Thanks @jwilliams-mongo, took all those suggestions & ready for another pass when you get a sec.

**Comment by zach-carr** [comment]
Hi @mmarcon, this connection page is ready for you to review whenever you have time. This is the page that we skipped tech review on to merge for a deadline, so please feel free to provide feedback on anything there. Thanks!

File: source/connect.txt [hunk 6]
```diff
@@ -34,43 +34,79 @@ This is equivalent to the following command:
...
 
-.. note:: Connecting to |service|
+  .. example:: 
+
+     To connect to a MongoDB
+     instance running on a remote host on port 28015:
+
+     .. code-block:: sh
+
+        mongosh --host mongodb://mongodb0.example.com:28015
```
Comment (mmarcon) [inline source/connect.txt:98]:
```suggestion
        mongosh --host mongodb0.example.com
```

File: source/connect.txt [hunk 7]
```diff
@@ -34,43 +34,79 @@ This is equivalent to the following command:
...
+
+     To connect to a MongoDB
+     instance running on a remote host on port 28015:
+
+     .. code-block:: sh
 
-   If your remote host is a |service-fullname| cluster, you can copy 
-   your connection string from the |service| UI. To learn more, see 
-   :atlas:`Connect to a Cluster <connect-to-cluster/#use-the-connect-dialog-to-connect-to-your-cluster>`.
+        mongosh --host mongodb://mongodb0.example.com --port 28015
```
Comment (mmarcon) [inline source/connect.txt:109]:
```suggestion
        mongosh --host mongodb0.example.com --port 28015
```

**Comment by zach-carr** [comment]
Thanks @mmarcon, ready for another look whenever you have a moment. 

I removed the middle bullet from [MongoDB Instance on a Remote Host](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker/DOCSP-10666/connect#mongodb-instance-on-a-remote-host) that mentioned the format --host <host>:<port>, which doesn't seem to work after all (I was testing with a URI rather than hostname).

**Comment by mmarcon** [comment]
I think the change looks good but in the staged view I still see this (`--host URI` instead of `--host hostname`):

image

**Comment by zach-carr** [comment]
Thanks @mmarcon, not sure why the staged view was outdated. It should be updated [here](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker/DOCSP-10666/connect#mongodb-instance-on-a-remote-host). I see:

screenshot

**Pull request merged**